backport/backport.ts

```diff
@@ -166,11 +166,13 @@ const getFailedBackportCommentBody = ({
 		'# Fetch latest updates from GitHub',
 		'git fetch',
 		'# Create a new working tree',
-		`git worktree add ${worktreePath} ${base}`,
+		`git worktree add ${worktreePath} origin/${base}`,
 		'# Navigate to the new working tree',
 		`cd ${worktreePath}`,
 		'# Create a new branch',
 		`git switch --create ${head}`,
+		'# Install dependencies for pre-commit hooks',
+		`yarn install --frozen-lockfile`,
 		'# Cherry-pick the merged commit of this pull request and resolve the conflicts',
 		`git cherry-pick --mainline 1 ${commitToBackport}`,
```

Comment:
```suggestion
		`git cherry-pick --x ${commitToBackport}`,
```

not sure what the mainline 1 does?

Reply:
@torkelo Do you want me to make this change as part of the PR?

Reply:
yes